### PR TITLE
Components usage

### LINKED ISSUE #1118: JSX: Replace remaining HTML in examples with JSX
There are places left where HTML was not replaced with JSX

### PR DESCRIPTION
**Pull request by anna-wro**
Using react components in the styleguide
* List - list icon color & type

Closes #1118

### CODE REVIEW

File: src/components/content-box/pages/content-box.jsx [hunk 1]
```diff
@@ -69,12 +70,9 @@ const examplePart1 = <ContentBox>
...
   </ContentBoxContent>
   <ContentBoxActions>
-    <div className="sg-overlayed-box">
-      <Avatar imgSrc="https://source.unsplash.com/64x64/?dog"/>
-      <div className="sg-overlayed-box__overlay">
-        <Sticker type={STICKER_TYPE.ANSWER}/>
-      </div>
-    </div>
+    <OverlayedBox
+      children={<Avatar imgSrc="https://source.unsplash.com/64x64/?dog"/>}
```
Comment (matikrk) [inline src/components/content-box/pages/content-box.jsx]: put as normal children, no in props
<OverlayedBox> XYZ </OverlayedBox>

File: src/components/toplayer/iframe-pages/splash_screen_toplayer.jsx [hunk 2]
```diff
@@ -31,11 +32,11 @@ const SmallSpacedTopLayer = () =>
       <link rel="stylesheet" href="../../../style-guide.css"/>
     </head>
     <body>
-      <div className="sg-overlay">
+      <Overlay>
```
Comment (matikrk) [inline src/components/toplayer/iframe-pages/splash_screen_toplayer.jsx]: As i remember Overlay don't have children

Comment (anna-wro) [inline src/components/toplayer/iframe-pages/splash_screen_toplayer.jsx]: They have

Comment (matikrk) [inline src/components/toplayer/iframe-pages/splash_screen_toplayer.jsx]: maybe i saw old version :) 
os this is ok

File: src/components/content-box/pages/content-box.jsx (hunk 1 above)
Comment (anna-wro) [inline src/components/content-box/pages/content-box.jsx]: fixed

**Pull request merged**